OSX/build/macports_ansible/compileMythfrontendAnsible.zsh

```diff
@@ -367,17 +367,19 @@ else
   fi
   cd $REPO_DIR/ansible
   export ANSIBLE_DISPLAY_SKIPPED_HOSTS=false
-  ANSIBLE_FLAGS="--limit=localhost  --ask-become-pass"
+  ANSIBLE_FLAGS="-l \"localhost\" -K"
```

Comment:
Please change to the full switches

Reply:
I believe the limit option does not use the "=" to specify the hosts, according to documentation.